include/crm/msg_xml.h

```diff
@@ -49,6 +49,7 @@ extern "C" {
  */
 
 #define PCMK_XA_CRM_FEATURE_SET             "crm_feature_set"
+#define PCMK_XA_DIGEST                      "digest"
```

Comment:
Since this is not in the CIB or otherwise user-facing, it should be internal. The patchset format is borderline; I would lean to saying the ability to create and apply patchsets is public (via crm_diff)  but the format itself is not guaranteed.

Reply:
Makes sense, I thought we gave the digest in some kind of public response but it looks like it's all in based-to-based XML (e.g., pings) or text-like outputs.

What's your verdict on PCMK_XA_FORMAT? We provide the format as an XML attribute in API responses because, in theory, a v2 format should always be the same. We even provide an XML schema (in xml/api) for the v2 diff format.

Reply:
Ah I forgot we made a public schema. It's not as important but I guess we can make it public, no changes are expected

Reply:
Yeah to be clear nothing uses the schema AFAIK. I provided it as an aid (not least for myself) because there's a lot going on in the diff output. Also ideally it should be parsable if someone does want to, since the format has been stable.